### ISSUE TITLE
MacOS darkmode debug pointer readability

### ISSUE THREAD

**Issue by Erikvdv** [QuestPDF/QuestPDF#999]
When in dark mode the code preview in debug pointer is hardly readable:

<img width="482" alt="Screenshot 2024-09-15 at 09 24 49" src="https://github.com/user-attachments/assets/b3ac4bd4-2a39-45ca-b7e6-279c0258c3b0">

BTW great feature!

**Comment by MarcinZiabek** [comment]
Thank you for your positive feedback and for noticing this issue 😄  

Could you please try the `2024.10.2` release?

**Comment by Erikvdv** [comment]
yes, this is fixed. Would be nice if the view could be broader as well as now I cannot read the whole line. (especially since opening the code is broken for me as mentioned in another issue)

**Comment by MarcinZiabek** [comment]
Could you please show how the code preview looks like in your case? 

I see two options:
1) Make the code preview wider. This will likely fix most cases. 
2) Detect the common indentation level along displayed lines and move all lines to the left. This may help but I also feel that indentation provides a hint where the code hint (how deeply nested).

**Comment by Erikvdv** [comment]
<img width="504" alt="Screenshot 2024-09-16 at 19 18 30" src="https://github.com/user-attachments/assets/098ba4c5-ecdc-4049-ab79-9a2d16c24549">

Here is an example

**Comment by MarcinZiabek** [comment]
Could you please upgrade to the `2024.10.3` release? 😄 

The version contains an improved source-code snippet for the selected item bar (the preview is wider, and the code is also scrollable).

**Comment by Erikvdv** [comment]
Yes, works now. Personally I prefer to have it a bit wider even, but at least it is workable now.

**Comment by v3nus-py** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by MarcinZiabek** [comment]
This bug appears to be already resolved.

Since there has been no recent activity, I’m closing this issue. If you experience a similar problem or have a related request, feel free to open a new GitHub issue. I appreciate your feedback! ❤️
